### ISSUE TITLE
Play in Viewport - build failed - Windows 8.1

### ISSUE THREAD

**Issue by zauberparacelsus** [armory3d/armory#22]
Getting a "Build Failed" error when I try to run Play in Viewport:

```
Read new prefs: C:\Users\Robert\AppData\Roaming\Blender Foundation\Blender\2.78\config\userpref.blend
AL lib: (EE) UpdateDeviceParams: Failed to set 44100hz, got 48000hz instead
found bundled python: C:\Program Files\Blender Foundation\Blender\2.78\python
read blend: C:\Users\Robert\Documents\ArmoryTest.blend
Scene built in 0.0020020008087158203
Creating Kha project.
Exporting asset 1 of 1 (blur_edge_pass_EnvCol_SSAO.arm).
Exporting asset 1 of 1 (compositor_pass_EnvCol_SSAO_CompoTonemap_CompoFXAA.arm).
Exporting asset 1 of 1 (deferred_EnvCol_SSAO.arm).
Exporting asset 1 of 1 (deferred_indirect_EnvCol_SSAO.arm).
Exporting asset 1 of 1 (deferred_light_EnvCol_SSAO.arm).
Exporting asset 1 of 1 (ssao_pass_EnvCol_SSAO.arm).
Exporting asset 1 of 1 (world_EnvCol_SSAO.arm).
Exporting asset 1 of 1 (brdf.png).
Exporting asset 1 of 1 (noise8.png).
Exporting asset 1 of 1 (Scene.arm).
Exporting asset 1 of 1 (World_irradiance.arm).
Exporting asset 1 of 1 (World_material.arm).
Exporting asset 1 of 1 (mesh_Cube.arm).
Exporting asset 1 of 1 (mesh_Plane.arm).
Exporting asset 1 of 1 (deferred_path_low.arm).
Compiling shader 1 of 1 (blur_edge_pass_EnvCol_SSAO.frag.glsl).
No shader compiler found.
internal/child_process.js:278
  var err = this._handle.spawn(options);
                         ^

TypeError: Bad argument
    at TypeError (native)
    at ChildProcess.spawn (internal/child_process.js:278:26)
    at Object.exports.spawn (child_process.js:372:9)
    at fs.stat (C:\Users\Robert\Downloads\Armory_p01_win32\Armory\armsdk\win32\resources\app\extensions\kha\Kha\Tools\khamake\out\ShaderCompiler.js:222:51)
    at FSReqWrap.oncomplete (fs.js:123:15)
 Build failed, check console
```

**Comment by luboslenco** [comment]
Were there any 'Path too long' errors when you unpacked? Can you search for 'krafix.exe' file in the sdk? That one is used for shader compiling.

**Comment by zauberparacelsus** [comment]
No path length errors that I saw.  That's a copy & paste of the entire output of the shell.  Did a search for krafix.exe in the entire unzipped folder, and no sign of it.  Maybe it's missing from the Win32 package?

**Comment by luboslenco** [comment]
Hm, should be there, got reports that it's working on Windows.

It's a bit deep:
`
.. armsdk/win32/resources/app/extensions/kha/Kha/Kore/Tools/krafix
`

**Comment by zauberparacelsus** [comment]
Hmm, I only see two files in that folder:  D3DCompiler_47.dll and dxwebsetup.exe

**Comment by luboslenco** [comment]
Odd, checking myself just now.

**Comment by zauberparacelsus** [comment]
Weird.  It's present in the zip archive

**Comment by zauberparacelsus** [comment]
I found the problem.  Norton is flagging it as a threat and removing it.

**Comment by luboslenco** [comment]
Oh my. Will have to note this in docs.

**Comment by johnbourne** [comment]
Hmm, I'm having the same problem now with the current (June 11) blender.  I restored loaded armory.py from github in as the addon (maybe the same as from the 2-20 version of armory3d with blender).  I then loaded from github the armsdk and pointed to it from the addon.  I ran the playground.blend .  I have an error in the console as shown below - the shader compiler seems to compile things fine and then it gives the error shown below.  the .../Tools folder seems to be missing ... 
I may have the same Norton problem that Zauderparacelsus mentions..  I'm not sure how to check to see what is missing.  I see several of the directories have ... "backup" appended - maybe that is from Norton... Simply too much to figure out.

Appreciate any help to get this going again.  Probably for me (and others) would be a Blender version integrated with armory.....I'm guessing luboslenco is waiting until final release.


Compiling shader 4 of 19 (Material_shadowmap.vert.glsl) failed:
No shader compiler found.
(node:15421) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 9): No shader compiler found.
(node:15421) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
child_process.js:383
    throw new TypeError('"file" argument must be a non-empty string');
    ^

TypeError: "file" argument must be a non-empty string
    at normalizeSpawnArguments (child_process.js:383:11)
    at Object.exports.spawn (child_process.js:496:38)
    at fs.stat (/Users/johnbourne/Documents/ArmoryBuild/armsdk/Kha/Tools/khamake/out/ShaderCompiler.js:333:55)
    at /Users/johnbourne/Documents/ArmoryBuild/armsdk/Kha/Tools/khamake/node_modules/graceful-fs/polyfills.js:284:29
    at FSReqWrap.oncomplete (fs.js:152:21)
Finished in 0.33591318130493164
